css/projects.css

```diff
@@ -0,0 +1,59 @@
+/*Projects CSS*/
+body {
+    color: black;
+    font-family: 'Langar', cursive;
+}
+
+header,
+footer {
+    background-color:   rgb(238, 237, 243);
+
+}
+
+a {
+    padding: 15px;
+    text-decoration: none;
+    color: black;
+}
+
+.header-text {
+    font-size: 25px;
+    margin-left: -80px;
```

Comment:
You're not using the grid system correctly here. Don't give the blue square its own column class - just put it inside the same `col-7` as the header text, so they can sit next to each more easily.